rspace/src/main/scala/coop/rchain/rspace/IStore.scala

```diff
@@ -4,6 +4,8 @@ import coop.rchain.rspace.internal._
 
 import scala.collection.immutable.Seq
 
+case class StoreSize(sizeOnDisk: Long, dataEntries: Long)
```

Comment:
Could we move this elsewhere?  I'd like this file to only contain the definition of `IStore`.  I'm fine with putting this in its own file for now.

Reply:
I don't think that it's good idea to separate interface and the "return value". Anyway extract it (and return back if needed) - just a single mouse click. Done.